### DISCUSSION TITLE
Accessing Rosstat Data

### DISCUSSION THREAD

**Discussion by larry77** [amattioc/SDMX#214] [Ideas]
Hello,
For professional reasons I would like to query Rosstat (the Russian equivalent of Eurostat).
From what I see here

https://www.oecd.org/sdd/44687401.pdf

and here

https://registry.sdmx.org/webservice/structure.html

(at the webpage above, in the "Agency" menu I see "[RU1] Rosstat")

It seems that Rosstat implements to some extent the SDMX standard. Would it be possible to add Rosstat to the list of data providers in RJSDMX?
Thanks a lot!

**Comment by amattioc** [comment]
Hi @larry77 , I do not see any registered dataflows for the RossStat agency:

https://registry.sdmx.org/ws/public/sdmxapi/rest/dataflow/RU1/all/latest/?format=sdmx-2.1&detail=full&references=none 

Furthermore, in the paper you sent I cannot find and URL for the russian web service. Do you have any other info?

Best, Attilio

**Comment by larry77** [comment]
Hi, unfortunately no more info for now.
